### PR TITLE
Added arm64-v8a ABI and binaries, update libpd module and fix ScenePlayer build

### PR DESCRIPTION
**Pull request by tkirshboim**
Hi Joe,

these are just a few other minor changes I did recently. I thought maybe you could merge it to your branch so that the [existing pull request](https://github.com/libpd/pd-for-android/pull/10) on the pd-for-android repo would simply be updated with these changes. Otherwise I could create a separate pull request once the existing one is merged.

Best,
Tal

### CODE REVIEW

File: ScenePlayer/res/layout/recording_player.xml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-	xmlns:rjdj="http://schemas.android.com/apk/res/me.rjdj.rjdjroid"
```
Comment (joebowbeer) [inline ScenePlayer/res/layout/recording_player.xml:3]: If xmlns is needed anywhere for something similar, it should point to the generic "res-auto" schema for gradle-build compatibility.

Comment (tkirshboim) [inline ScenePlayer/res/layout/recording_player.xml:3]: Right. In this case it's not used at all in that file, that's why I removed
it.

File: PdCore/jni/libpd [hunk 2]
```diff
@@ -1 +1 @@
-Subproject commit c55a675319e7fec3663a49b9ccbc1a77db1d4ee7
```
Comment (joebowbeer) [inline PdCore/jni/libpd:1]: I just noticed that I missed this part, which updates libpd to 0.8.3

**Comment by joebowbeer** [comment]
Should my pull request #10 simply remove the libs binaries? (As it automates the ndk-build?)

**Comment by tkirshboim** [comment]
I would keep the binaries there for now to not break stuff for people that
might want to simply update their existing pd-for-android dependency
without moving to gradle and setting up the ndk. I think we can remove them
when as soon as we provide downloadable .aar of pd-for-android (either on a
maven repo or another way)

**Comment by tkirshboim** [comment]
changes manually added to [gradle_mods branch](https://github.com/joebowbeer/pd-for-android/tree/gradle_mods) - closing

**Pull request closed without merging**